README.md

```diff
@@ -162,7 +162,8 @@ attributeFields(Model, {
   exclude: [], // array of model attributes to ignore - default: []
   only: [], // only generate definitions for these model attributes - default: null
   globalId: true, // return an relay global id field - default: false
-  map: {} // rename fields - default: {}
+  map: {}, // rename fields - default: {}
+  allowNull: true, // automatically wrap mandatory fields in `GraphQLNonNull` - default: true
```

Comment:
The description here is wrong.
Setting allowNull does not wrap mandatory fields, it disabled wrapping, and the default is false.